docs/reference/server-worker-configuration.md

```diff
@@ -48,36 +57,38 @@ There must be a configuration for each platform the worker will be run on.
 
 This is an example excerpt of a `localDeployment` section from the main config file:
 ```json
+      "windows": {
+        "localBuildDirectory": "bin/x64/ReleaseWindows",
+        "command": "...",
+        "arguments": [
+          "..."
+        ]
+      },
       "macos": {
         "localBuildDirectory": "bin/x64/ReleaseMacOS",
-        "command": ...,
+        "command": "...",
         "arguments": [
-          ...
+          "..."
         ]
       },
       "linux": {
         "localBuildDirectory": "bin/x64/ReleaseLinux",
-        "command": ...,
+        "command": "...",
         "arguments": [
-          ...
-        ]
-      },
-      "windows": {
-        "localBuildDirectory": "bin/x64/ReleaseWindows",
-        "command": ...,
-        "arguments": [
-          ...
+          "..."
         ]
       }
 ```
 
-The platform name can be one of `macos`, `linux` or `windows`. Each of these can have the following fields:
+The platform name can be one of `windows`, `macos` or `linux`. Each of these can have the following fields:
 
 | Field | Required/Optional | Description | 
 | :------------- | :------------- | :------- |
-| `localBuildDirectory` | Required | The directory containing the binary to be executed. |
+| `localBuildDirectory` | Required | The directory containing the binary to be executed. <br><br> **Note:** Your built out worker must be self-contained to the `localBuildDirectory` and all files that your worker might depend on while running must be present in this folder. Files outside of this folder will not be uploaded to the cloud as part of your project assembly. |
 | `command` | Required | The base command to be executed to start the worker. |
-| `arguments` | Required | A list of arguments to provide to the base command. The arguments can contain the same placeholder strings as specified in the previous version of configuration, which are interpolated by SpatialOS before the worker is launched. See the [worker launch configuration](https://docs.improbable.io/reference/latest/shared/worker-configuration/launch-configuration) documentation for more information. |
+| `arguments` | Required | A list of arguments to provide to the base command. The arguments can contain the same placeholder strings, like `{IMPROBABLE_RECEPTIONIST_HOST}`, which are interpolated by SpatialOS before the worker is launched. See our documentation on [worker launch configurations](https://docs.improbable.io/reference/latest/shared/worker-configuration/launch-configuration) for more information. |
```

Comment:
It might be my lack of context, but I'm not sure what "the same" refers to here - could you clarify?

Reply:
Yeah, this is unclear. I changed the sentence.